src/NosCore.PathFinder.Gui/GuiWindow.cs

```diff
@@ -121,20 +122,30 @@ protected override void OnUpdateFrame(FrameEventArgs e)
         {
             base.OnUpdateFrame(e);
 
-            if (Keyboard[Key.Escape])
+            if (KeyPress(Key.Escape))
             {
                 Exit();
             }
         }
 
+        KeyboardState keyboardState, lastKeyboardState;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Declare 'lastKeyboardState' in a separate statement.](https://app.codacy.com/app/NosCoreIO/NosCore/pullRequest?prid=4456396)